project/prepare-release.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -e
+
+# NOTES:
+#     To delete a tag use: git push origin :v1.0.0
+
+if [[ $1 != "patch" && $1 != "minor" && $1 != "major" ]] ; then
+  echo "    Bad input: \"$1\". Expected an update type of \"patch\", \"minor\" or \"major\"";
+  exit 1;
+fi
+
+echo ""
+echo ""
+echo "Publishing Propel $PACKAGE_VERSION"
+echo ""
+echo ""
+echo "Building Library"
+echo ""
+gulp default --throw-error
+
+echo ""
+echo ""
+echo "Building Docs"
+echo ""
+esdoc -c esdoc.json
+
+echo ""
+echo ""
+echo "Update NPM Version"
+echo ""
+npm --no-git-tag-version version $1
```

Comment:
At this point it's unclear why you wouldn't want a version commit and tag, but assume it's because you want to manually handle this in 68.